### ISSUE TITLE
Using the framework over SIM800L  PPPoS or TCP connection ...

### ISSUE THREAD

**Issue by MKoroschetz** [tve/mqboard#33]
This is not an issue. My apologies if I ask my question here.

I found your very nice framework which covers the needs for a project I am working on. The specific question I would like to ask, if it would work using a  a SIM800L GPRS data connection, instead of WiFi.
My project is currently working with the @loboris [MP build](https://github.com/loboris/MicroPython_ESP32_psRAM_LoBo) which is stale since 2018. I am facing some problems which require me to move to a more updated build. As Hardware backbone I am using the [T-Call-SIM800](https://github.com/Xinyuan-LilyGO/LilyGo-T-Call-SIM800). 
Some of my problems are related to the RTC in the ESP32, which you have addressed in your build, and some other fixes I am sure would be beneficial as well.
Basically, I am not sure if your [v1.12-tve2](https://github.com/tve/micropython/releases/tag/v1.12-tve2) or 1.13 build with your MQTT implementation will work with the underlying connection established by the SIM800L. It is my understanding that the @lobo implementation implies C level integration to make it work over PPPoS. Not sure if your build does something similar or assumes to have a full TCP/IP stack available as is the case with WiFi.
I sincerely would appreciate your insight and suggestions to this question, as I am excited to try out your framework. 
Thanks in advance.
